Add python class for 'RobotTrajectory' ROS message type manipulation

karolyartur opened the pull request:
### Description

Added a python class to achieve the same functionality as in 'moveit_core/robot_trajectory' module.

Currently, it supports setting, getting and appending trajectories, but this functionality can be expanded.

To use the class in python code use
```
from moveit_commander import RobotTrajectoryCommander
```
after building and re-sourcing the workspace.

felixvd committed:
Add collision setting to PlanningScene

felixvd committed:
Add subframes to Rviz motion planning text

Commit:
Add parameter to getJacobian Python interface

karolyartur committed:
Add 'RobotTrajectory' manipulation functionality

karolyartur committed:
Include 'robot_trajectory.py' in '__init.py__'

felixvd commented on moveit_commander/src/moveit_commander/robot_trajectory.py:
```diff
@@ -0,0 +1,51 @@
+# Software License Agreement (BSD License)
+#
+# Copyright (c) 2012, Willow Garage, Inc.
```
Please update

felixvd commented on moveit_commander/src/moveit_commander/robot_trajectory.py:
```diff
@@ -0,0 +1,51 @@
+#    copyright notice, this list of conditions and the following
+#    disclaimer in the documentation and/or other materials provided
+#    with the distribution.
+#  * Neither the name of Willow Garage, Inc. nor the names of its
```
Here, too

felixvd reviewed (commented):
Can you also add a test for this function?

karolyartur committed:
Update copyright

karolyartur committed:
Fix 'append' function

Fixed the timing of the appended trajectory.
Use copies, so the class does not modify the trajectory objects
that were passed to its member functions as parameters.

karolyartur committed:
Add test for 'RobotTrajectoryCommander' functionality

karolyartur committed:
Add test file

karolyartur committed:
Make script executable

karolyartur committed:
Add test to CMakeLists
